code/modules/research/xenoarchaeology/finds/finds_splinter.dm

```diff
@@ -0,0 +1,58 @@
+/**
+	A shard of supermatter
+	Touch it - You lose the hand
+	It hit you - You lose the limb it hit
+	It hit a supermatter shard - It increases its mass
+		3 splinters collide with a supermatter shard - It's now a crystal. DON'T FUCK IT UP
+	It gets hit with an explosion - EMP and more explosion
+
+**/
+
+/obj/item/supermatter_splinter
+	name = "supermatter splinter"
+	desc = "A superdense chunk of supermatter. It hums ever so slightly, with swirls of semi-absorbed matter orbiting it. <b>It doesn't look very safe to touch.</b>"
+	icon = 'icons/obj/shards.dmi'
+
+/obj/item/supermatter_splinter/New()
+	..()
+	name = "supermatter [pick("splinter","chunk","core")]"
```

Comment:
This and the following are independant, which means you can get a small supermatter core and a large supermatter chunk. Is that intended ?

Reply:
The thing below is the icon state though